packages/fxa-settings/src/components/Banner/index.stories.tsx

```diff
@@ -7,10 +7,12 @@ import Banner, { BannerType } from '.';
 import { Meta } from '@storybook/react';
 import AppLayout from '../AppLayout';
 import { Subject } from './mocks';
+import { withLocalization } from '../../../.storybook/decorators';
 
 export default {
-  title: 'components/Banner',
+  title: 'Components/Banner',
```

Comment:
I don't really have an opinion here and its awesome sorting is fixed. It seems like before this was done to match the folder name exactly. For example the `components` folder is lower case. I don't see why this would be an issue though. I feel like what you have here feels more consistent 👍 .

Reply:
We had a mix of both previously, which was breaking the sorting. Could have gone either way for `components` as it's rendered in all uppercase in the storybook list anyway. 

![image](https://user-images.githubusercontent.com/22231637/219509950-4f9c8694-197c-4a84-b731-4f324959cdaa.png)

Reply:
We should eventually get to the point where we don't need the `title` property, but at the moment our folder organization doesn't fully reflect the components/pages organization that we have in the storybook. 

This ticket (https://mozilla-hub.atlassian.net/browse/FXA-6465) would need to be completed before we can consider removing the `title` property for stories. I'll add a comment on that ticket.